i18n/fr/docusaurus-plugin-content-docs/current/full-node/Full-node-on-raspberry-pi.md

```diff
@@ -0,0 +1,161 @@
+---
+sidebar_position : 30
+titre : Full Node sur Raspberry Pi
```

Comment:
here too

Reply:
fixed in https://github.com/alephium/wiki/pull/39/commits/aa98c0da5a79c88cbe3ac669aeaa6acb4a6c3f8f